assets/sass/v2/_okta-verify.scss

```diff
@@ -49,6 +49,8 @@
 }
 
 .okta-verify-uv-callout-content {
+  margin-bottom: 15px;
```

Comment:
since we are adding a new css.. can we use `em` instead of pixels

Reply:
everything else in this file uses 15px so i wanted to keep that consistent